lib/operation.js

```diff
@@ -57,12 +43,12 @@ const is = require('./is');
  * @throws {Error} Invalid parameters
  */
 function rotate (angle, options) {
-  if (this.options.useExifOrientation || this.options.angle || this.options.rotationAngle) {
+  if (!is.defined(angle)) { return this.autoOrient(); }
+
+  if (this.options.angle || this.options.rotationAngle) {
     this.options.debuglog('ignoring previous rotate options');
```

Comment:
This warning _says_ that previous rotations will be ignored, but that isn't actually true unless you reset `options.angle` and `options.rotationAngle` to `0` here. Unless you do, these two sets of commands will produce a different output:

```
sharp(img)
  .rotate(90) // cardinal angle
  .rotate(5)
// output will be rotated 95 degrees

sharp(img)
  .rotate(91) // non-cardinal angle
  .rotate(5)
// output will be rotated 5 degrees
```

Want me to throw in a fix for that? Or alter the warning? Or just leave as is?

```suggestion
    this.options.debuglog('ignoring previous rotate options');
    this.options.angle = 0;
    this.options.rotationAngle = 0;
```

Reply:
Yes please to fixing this at the same time.